src/test/java/client/PetClient.java

```diff
@@ -74,47 +74,47 @@ public static Pet postPet(Pet petToPost) throws IOException {
 
     }
 
-    public static Pet getPetById() throws IOException {
+    public static Response getPetById(long petId, int statusCode) throws IOException {
 
         LOGGER.debug("sending request");
-        RequestSpecification res = given()
+        RequestSpecification res = RestAssured.given()
             .spec(buildReq());
 
         LOGGER.debug("expecting response");
-        Pet pet = res
+        Response pet = res
             .when()
-            .get(Resources.getPetById("669118"))
+            .get(Resources.getPetById(petId))
             .then()
+            .statusCode(statusCode)
             .spec(buildRes())
             .log()
             .body()
-            .body("status", equalTo("available"))
             .extract()
-            .response()
-            .as(Pet.class);
+            .response();
+//            .as(Pet.class);
```

Comment:
Why did you comment that line?

Reply:
@andyyarych I put it it the getPet tests because it uses different classes